### PR TITLE
Deploy LIFI to Scroll

### PR DESCRIPTION
**Pull request by ezynda3**

### CODE REVIEW

File: config/stargate.json [hunk 1]
```diff
@@ -148,6 +151,10 @@
     {
       "chainId": 1101,
       "lzChainId": 158
+    },
+    {
+      "chainId": 534352,
+      "lzChainId": 214
     }
```
Comment (maxklenk) [inline config/stargate.json:158]: @ezynda3 did you add the new id to the mappings on all all other chains supported by stargate?

File: deployments/scroll.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  "WithdrawFacet": "0xf600Caa1468FFaa0B8Eef877A84152e2bDdDe2Fb",
+  "DexManagerFacet": "0xbd0B19349A4D0DF5Abb67D31659d4C5C17b78901",
+  "AccessManagerFacet": "0xdbffD7A135b332748Df2784D214c5f48eC706C3a",
+  "PeripheryRegistryFacet": "0x36eacDF8e9CF209d84B427316839800a81a2c45A",
+  "GenericSwapFacet": "0x3912De64539B960166940F2BCD55723f56A7358D",
+  "StandardizedCallFacet": "0x9D4B9784fC46496926ee4784e490F14d106D83a8",
+  "CalldataVerificationFacet": "0x9157b6550D1C3209C20aCAd12A2dC57Df34Ca48b",
+  "SymbiosisFacet": "0xb8C5408ED5f40ca8df929A50DD36b252B6b39f76",
+  "CBridgeFacet": "0xc31F4392EE89d81E7303e2c1CDE210414F39E2ea",
+  "CBridgeFacetPacked": "0x4f62E7a939C62316F76E2afC644A9A907Dc30D2d",
```
Comment (maxklenk) [inline deployments/scroll.json:15]: @ezynda3 please also make sure the approval scripts for Packed Facets have been executed. I think it is missing here:
https://scrollscan.com/address/0x4f62E7a939C62316F76E2afC644A9A907Dc30D2d

File: config/stargate.json (hunk 1 above)
Comment (ezynda3) [reply config/stargate.json:158]: Yes

File: deployments/scroll.json (hunk 2 above)
Comment (ezynda3) [reply deployments/scroll.json:15]: done

**Pull request merged**